### DISCUSSION TITLE
gradient start and end diffrently

### DISCUSSION THREAD

**Discussion by cbsa100** [tomik23/circular-progress-bar#71] [Q&A]
i need a progress bar that start red and finish green
when doing that using the current gradient option, i get a circle that start red, transform to green and then back to black.
how can i make it go only 1 way?

also how can the end of the gradient be relative to the cut prop?

**Comment by tomickigrzegorz** [comment]
Unfortunately this library doesn't support circle gradient but only linear gradient, you probably need something like [circular-progress-indicator-with-a-color-gradient-with-svgs](https://stackoverflow.com/questions/61363990/circular-progress-indicator-with-a-color-gradient-with-svgs) or even build a circle using conic-gradient (css).
Maybe in the future it will be possible to get a cicle gradient, but at the moment it is not possible.
